### ISSUE TITLE
Next js rewrite proxy

### ISSUE THREAD

**Issue by PhilGarb** [PostHog/posthog#14999]
## Bug description

I am trying to use next js rewrites https://nextjs.org/docs/api-reference/next.config.js/rewrites to proxy posthog calls. While it does work for most cases the recorder.js fails for me with this setup.

*If you are on PostHog Cloud it would be really valuable if you can share any links where the problem occurs. This speeds up our ability to troubleshoot tremendously.* 

## How to reproduce

Please see the attached codesandbox => https://codesandbox.io/p/sandbox/tender-poincare-2x9imi?file=%2Fnext.config.js
1. Add a valid posthog token to .env.
2. Run npm dev
3. open the page and see that the request to `login?next=/static/recorder-v2.js/%3Fv%3D1.52.0` fails.

## Environment

- [ ] PostHog Cloud US, project ID: [please provide from https://app.posthog.com/project/settings#project-id]
- [x] PostHog Cloud EU, project ID: 2957
- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see ["Sunsetting Kubernetes support"](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: 1.52.0

**Comment by benjackwhite** [comment]
Firstly - thanks for the awesome bug report! Codesandbox and everything _**:chefs_kiss:**_ 😅 

Took a little bit of effort to figure out (mostly because I forgot you have to restart Next.js when changing the config...) _but_ I'm pretty sure it's due to the trailing slash in the config. For the API endpoints it works fine but for files like the js file a trailing slash makes no sense.

```
// ❌
{
  source: "/ingest/:path*",
  destination: "https://eu.posthog.com/:path*/",
}

// ✅
{
  source: "/ingest/:path*",
  destination: "https://eu.posthog.com/:path*",
}

**Comment by benjackwhite** [comment]
Also I will be adding this to the docs ASAP.

**Comment by PhilGarb** [comment]
Ahh fantastic :) I will give it a shoot. Next.js actually just released the reloading on config change with 13.3 last week. Just a moment to early with this one :)
